docs/frameworks/symfony.md

```diff
@@ -202,33 +156,47 @@ framework:
 If using Webpack Encore, you also need to add the following config at the end of `webpack.config.js`
 
 ```js
+// webpack.config.js
+
 if (Encore.isProduction()) {
     // Note the '/build' at the end of the URL
     Encore.setPublicPath('https://<bucket-name>.s3.amazonaws.com/build');
     Encore.setManifestKeyPrefix('build/')
 }
 ```
 
-Finally, you can compile assets for production in the `public` directory, then synchronize that directory to a S3 bucket:
+Then, you can compile assets for production in the `public` directory, and synchronize that directory to a S3 bucket:
 
 ```bash
 php bin/console assets:install --env prod
 # if using Webpack Encore, additionally run
 yarn encore production
-aws s3 sync public/ s3://<bucket-name>/ --delete --exclude index.php
+aws s3 sync public/ s3://<bucket-name>/ \
+  --delete \
```

Comment:
If you delete stuff, it would mean that there is some downtime for your application. (ie no assets)

Reply:
True, I think the solution is to sync twice. First without `--delete` and then with the `--delete` option.

Or do you have another solution ?

Reply:
I'll just leave the old files :)